src/store/getters/state-getters.ts

```diff
@@ -401,15 +401,16 @@ export const hasFirmChanged = (state: StateIF): boolean => {
   return (
     hasBusinessNameChanged(state) ||
     hasNatureOfBusinessChanged(state) ||
-    officeAddressesChanged(state)
+    officeAddressesChanged(state) ||
+    hasOrgPersonChanged(state)
```

Comment:
I suggest:
- rename officeAddressesChanged -> hasOfficeAddressesChanged
- rename hasOrgPersonChanged -> hasPeopleAndRolesChanged

Up to you though.

Reply:
Updated!